tests/nemo_text_processing/zh/data_text_normalization/test_cases_cardinal.txt

```diff
@@ -1,161 +1,84 @@
 10000~一万
-负10000~负一万
-正100000~正十万
+负1000~负一千
+100000~十万
 210~二百一十
-负200~负二百
-负219~负二百一十九
+200~二百
+219~二百一十九
 5000~五千
 5100~五千一百
 5110~五千一百一十
 5111~五千一百一十一
 5010~五千零一十
-5099~五千零九十九
+5010~五千零一十
 5001~五千零一
-5,000~五千
-5,100~五千一百
-5,010~五千零一十
-5,001~五千零一
-50,000~五万
-51,000~五万一千
-50,100~五万零一百
-50,010~五万零一十
-51,100~五万一千一百
-51,110~五万一千一百一十
-50,011~五万零一十一
-50,001~五万零一
-50,010~五万零一十
 50000~五万
 51000~五万一千
 50100~五万零一百
 50010~五万零一十
-50001~五万零一
 51100~五万一千一百
 51110~五万一千一百一十
-51111~五万一千一百一十一
-50111~五万零一百一十一
 50011~五万零一十一
-500,000~五十万
-510,000~五十一万
-501,000~五十万一千
-500,100~五十万零一百
-500,010~五十万零一十
-500,111~五十万零一百一十一
-501,100~五十万一千一百
-501,111~五十万一千一百一十一
-511,111~五十一万一千一百一十一
-100,000~十万
-110,000~十一万
+50001~五万零一
+50010~五万零一十
+50000~五万
+51000~五万一千
+50100~五万零一百
+50010~五万零一十
+50001~五万零一
+51100~五万一千一百
+50011~五万零一十一
+500000~五十万
+510000~五十一万
+501000~五十万一千
+500100~五十万零一百
+100000~十万
+110000~十一万
 500000~五十万
 510000~五十一万
 501000~五十万一千
 500100~五十万零一百
 500010~五十万零一十
-500001~五十万零一
-500111~五十万零一百一十一
-501100~五十万一千一百
 501111~五十万一千一百一十一
 511111~五十一万一千一百一十一
 100000~十万
 110000~十一万
 1100000~一百一十万
-1010000~一百零一万
-1001000~一百万一千
-1000100~一百万零一百
-1000010~一百万零一十
 1000001~一百万零一
 1000000~一百万
-1,000,000~一百万
-1,100,000~一百一十万
-1,010,000~一百零一万
-1,001,000~一百万一千
-1,000,100~一百万零一百
-1,000,010~一百万零一十
-1,000,001~一百万零一
 10000000~一千万
 11000000~一千一百万
 10100000~一千零一十万
 10010000~一千零一万
+11000000~一千一百万
+10100000~一千零一十万
+10010000~一千零一万
 10001000~一千万一千
-10010100~一千零一万零一百
-10010010~一千零一万零一十
-10010001~一千零一万零一
-10,000,000~一千万
-11,000,000~一千一百万
-10,100,000~一千零一十万
-10,010,000~一千零一万
-10,001,000~一千万一千
-10,010,100~一千零一万零一百
-10,010,010~一千零一万零一十
-10,010,001~一千零一万零一
-101111111~一亿一百一十一万一千一百一十一
+101111111~一亿零一百一十一万一千一百一十一
 110111111~一亿一千零一十一万一千一百一十一
 111011111~一亿一千一百零一万一千一百一十一
 111101111~一亿一千一百一十万一千一百一十一
 111110111~一亿一千一百一十一万零一百一十一
-111111011~一亿一千一百一十一万一千零一十一
-111111101~一亿一千一百一十一万一千一百零一
-111111111~一亿一千一百一十一万一千一百一十一
-101,111,111~一亿一百一十一万一千一百一十一
-110,111,111~一亿一千零一十一万一千一百一十一
-111,011,111~一亿一千一百零一万一千一百一十一
-111,101,111~一亿一千一百一十万一千一百一十一
-111,110,111~一亿一千一百一十一万零一百一十一
-111,111,011~一亿一千一百一十一万一千零一十一
-111,111,101~一亿一千一百一十一万一千一百零一
-111,111,110~一亿一千一百一十一万一千一百一十
-111,111,111~一亿一千一百一十一万一千一百一十一
+101111111~一亿零一百一十一万一千一百一十一
+110111111~一亿一千零一十一万一千一百一十一
+111011111~一亿一千一百零一万一千一百一十一
+111101111~一亿一千一百一十万一千一百一十一
 1011111111~十亿一千一百一十一万一千一百一十一
-1101111111~十一亿一百一十一万一千一百一十一
-1110111111~十一亿一千零一十一万一千一百一十一
+1101111111~十一亿零一百一十一万一千一百一十一
 1111011111~十一亿一千一百零一万一千一百一十一
-1111110111~十一亿一千一百一十一万零一百一十一
-1111111101~十一亿一千一百一十一万一千一百零一
-1111111111~十一亿一千一百一十一万一千一百一十一
-1,011,111,111~十亿一千一百一十一万一千一百一十一
-1,101,111,111~十一亿一百一十一万一千一百一十一
-1,110,111,111~十一亿一千零一十一万一千一百一十一
-1,111,011,111~十一亿一千一百零一万一千一百一十一
-1,111,101,111~十一亿一千一百一十万一千一百一十一
-1,111,110,111~十一亿一千一百一十一万零一百一十一
-1,111,111,011~十一亿一千一百一十一万一千零一十一
-1,111,111,101~十一亿一千一百一十一万一千一百零一
-1,111,111,110~十一亿一千一百一十一万一千一百一十
+11000000000~一百一十亿
+10000100000~一百亿零十万
 11000000000~一百一十亿

```

Comment:
why were some tests cases removed? are they still supported?

Reply:
I was hoping to reduce test time by removing some of them. They are still being supported.